iTechArt.Hotels.Database/Accounts.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[Accounts]
+(
+    [Id] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [Email] VARCHAR(40) NOT NULL, 
```

Comment:
Are you sure that 40 characters will be enough for any email?

Reply:
Had changed to 125, email lenght best practices says it should be between 50 and 125